### ISSUE TITLE
Allow a single empty line between jsdoc and codeblock

### ISSUE THREAD

**Issue by redaxmedia** [gajus/eslint-plugin-jsdoc#482]
Hey,

I wonder why this plugin is not supporting a single empty line after the jsdoc:

```
/**
 * @return string
 */

function foo()
{
    return 'bar';
}
```

At the moment only docked jsdocs like this are detected:

```
/**
 * @return string
 */
function foo()
{
    return 'bar';
}
```

A similar TSLint rule named [completed-docs](https://palantir.github.io/tslint/rules/completed-docs/) for example is okay with that.

**Comment by brettz9** [comment]
Please clarify what you mean by not supporting a single empty line.

We have a rule, `newline-after-description`, that can disallow (or require) a single empty line if that is the style chosen, but it is not required.

Our "recommended" config sets this rule by default to *require* a newline. You can change its value to "never" to prevent newlines, or just disable the rule `"jsdoc/newline-after-description": "off"` if you wish to cancel the rule if inherited from elsewhere.

**Comment by redaxmedia** [comment]
We are talking about different things - my question was not related to a newline after the description, it was related to a newline after the dockblock / jsblock...

A rule like that would be something like `"jsdoc/newline-after-jsdoc": "on"`.

**Comment by brettz9** [comment]
Gotcha... On the main [README](https://github.com/gajus/eslint-plugin-jsdoc), I've now added formal documentation for the preexisting `maxLines` and `minLines`settings (usable in conjunction with all rules, and enforceable via the `require-jsdoc` rule). (They were in the example code previously but with no dedicated section to explain their usage.)

So if you set the `maxLines` to at least 2, you should be good (and if you want it never be less than one empty line, set the `minLines` to 2 as well).

**Comment by redaxmedia** [comment]
Thank you :+1:
